.github/scripts/check_content.py

```diff
@@ -50,16 +51,6 @@ def check_file(relative_path: str, file: str) -> int:
     return errors
 
 
-def get_root_of_vault() -> str:
```

Comment:
Am very happy for this to be moved to utils.py, as it's more obvious there, and more likely to be spotted!

I think that the test I added for it should ideally be updated, as it was in IIRC `test_check_content.py` which isn't right now.... I think it should now be in `test_utils.py`???

Reply:
I have no clue how the tests work, so I'll trust your judgment!

Reply:
> I have no clue how the tests work, so I'll trust your judgment!

I was more trying to understand if you wanted to learn a bit about testing?